ngsa-asb-dev-eastus.env

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Please remove this file. These values should not be shared publicly. The environment variables will be set locally as the user goes through the implementation steps. in the Readme.md file, update the line where the AKS version is set, using the value in this file (ASB_K8S_VERSION='1.23.8'). The aks version needs to be updated because the version listed in the readme is not supported in some regions.

Reply:
Removed via a commit